src/runtimes/node/bundlers/esbuild/bundler_target.ts

```diff
@@ -35,20 +27,22 @@ const getModuleFormat = async (
   extension: string,
   configVersion?: string,
 ): Promise<{ includedFiles: string[]; moduleFormat: ModuleFormat }> => {
-  if (extension === ModuleFileExtension.MJS && featureFlags.zisi_pure_esm_mjs) {
+  if (featureFlags.zisi_pure_esm_mjs && extension === ModuleFileExtension.MJS) {
```

Comment:
nit: Should this be moved to the `if` block below?

Reply:
These are two different FF.

Reply:
Ah, my bad!